### ISSUE TITLE
Change the "confirm receipt" modal into a full page form

### ISSUE THREAD

**Issue by diegoliberman** [instedd/cdx#1567]

**Comment by straight-shoota** [comment]
> After accessing the form, the focus is placed automatically into the empty field for entering a sample ID.

I'm skipping this because the first field is *Institution*. It would be weird to autofocus the sample search field.

I postponed autocomplete. It should be simple to add in later, but isn't necessary for a MVP. You can do a partial search (press enter) after entering an unambiguous number of characters from the UUID.

**Comment by straight-shoota** [comment]
I think this needs some refinements to reflect the intended updates for transfer packages and boxes.

With #1582, I expect the typical workflow would *not* start with going to a `conform receipt` action and starting to individually confirm sample by sample.
Instead, when you receive a shipment, you would go to the transfers page, select the pending transfer package that you want to confirm and do the confirmation there. The confirmation happens in the context of the transfer package, not independently for every single transferred sample. What exactly that confirmation workflow entails is to be determined.

The transfer package view would show the contents of the package. Thus, you can directly see which items are expected to be present in the (physical) shipment.

First question is how find grained the confirmation of individual contents in a transfer package needs to be represented in CDX. Of course, the local lab technician needs to verify the actual shipping contents against the delivery note. But maybe that could happen outside of CDX and they just confirm receipt of the entire package.

I understand that a feature for scanning the labels of the package contents could help with the verification process. You just scan every item in the package and then everything is green.
When there is no scanner, maybe it's not necessary to involve an advanced confirmation step like inserting the last digits of the UUID. Instead, it could be sufficient to just tick a checkbox to account for the sample.

There are also some edge cases which apparently haven't been addressed before, but become more relevant with the process based on transfer packages:
* What happens when contents are missing from the shipment?
* What happens when there are more contents than documented?
* Can the contents of a transfer package be split across multiple physical shipments? Should you be able to confirm partially?

**Comment by diegoliberman** [comment]
@straight-shoota all valid points and questions.  

I understand that an option is to implement directly the confirmation of a transfer package.   But I'm worried about such a strategy, as we might run out of time and finish with a non-working end to end solution.  

Considering that the confirm receipt of individual samples is easy to implement (right?) I think it's safer to finish this flow, and then begin the new one independently.  That way, no matter what, at least there will be an option to transfer individual samples end to end.

Regarding your questions, my answer is that we won't deal with unhappy paths in the first stage of the transfer package feature.

**Comment by diegoliberman** [comment]
OK, sounds fine, let's go in the transfer package direction.  I'm not sure when we will have designs for that flow.  Hopefully you can move forward without the designed screens? I imagine a view with 1 row per transfer, and you can click in the row to see the detail of the transfer with the list of samples.

**Comment by straight-shoota** [comment]
Yeah exactly that's what I was aiming for. It's pretty straightforward.

**Comment by diegoliberman** [comment]
@straight-shoota does it make sense to you if we close/cancel this issue? Perhaps we can create a new breakdown with the final goals that we have for the transfer feature?

**Comment by diegoliberman** [comment]
Cancelling this one in favor of #1611.
